.github/workflows/run-tests.yml

```diff
@@ -244,13 +252,13 @@ jobs:
       - name: Post the link to the report
         id: post_report_link
         timeout-minutes: 60
-        if: always() && steps.put_report.outcome == 'success'
+        if: always() && steps.put_report.outcome
```

Comment:
Why? If it fails this link is irrelevant.

Reply:
My mistake. Reverted it back.